### DISCUSSION TITLE
deb.postinst script

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/nfpm#419] [Q&A] [answered]
- what would the postinst script for `deb` package look like?
- how can I take a look?
- is it somehow in the following format?

```sh
#! /bin/sh

set -e

case "$1" in
    configure)
        if [ -z "`getent group lpadmin`" ]; then
            addgroup --system lpadmin
        fi
    ;;

    abort-upgrade|abort-remove|abort-deconfigure)

    ;;

    *)
        echo "postinst called with unknown argument \`$1'" >&2
        exit 0
    ;;
esac

#DEBHELPER#

exit 0
```

see https://sources.debian.org/src/cups/2.3.3op2-7/debian/cups-client.postinst/

thx

**Comment by suntong** [comment, marked as answer]
Yes, as long as I provide the postinstall script like that.

**Reply by djgilcrease** [reply]
https://nfpm.goreleaser.com/tips/
